starwars/src/App.js

```diff
@@ -1,17 +1,48 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import People from './components/People/People';
 import './App.css';
 
 const App = () => {
   // Try to think through what state you'll need for this app before starting. Then build out
   // the state properties here.
+  const [people, setPeople] = useState([]);
 
   // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a 
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
+  useEffect(() => {
+    // implement axios allowing to retrieve data from API
+    axios
+      .get("https://swapi.co/api/people/")
+      .then(res => {
+        console.log(res.data.results);
+        setPeople(res.data.results);
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }, []);
 
   return (
     <div className="App">
       <h1 className="Header">React Wars</h1>
+      <h2 className="sub-header">Key Star Wars Personel</h2>
+      <div className="container">
+      {
+        people.map((person, index) => {
+          return (
+            <People 
+              key={index} 
+              person={person.name} 
```

Comment:
Here you're passing it 4 different props for the 4 different properties you want to display.  My approach would be to pass down the entire person object and just use dot/bracket notation in the child component.  That way would be more OOP (object oriented programming) and also fewer lines of code.